### ISSUE TITLE
Update is needed for the example "EchoXcode" for secure Service

### ISSUE THREAD

**Issue by shengyang998** [grpc/grpc-swift#236]

**Comment by shengyang998** [comment]
Concretely, at line 115 in file `EchoViewController.swift`
```swift 
service = Echo_EchoServiceClient(address: address, certificates: certificates, host: host)
```
Swift compiler complains:
```
Argument labels '(address:, certificates:, host:)' do not match any available overloads
```

**Comment by MrMage** [comment]
Thanks for the info! Could you file a pull request?

**Comment by MrMage** [comment]
This was fixed by #274.
